back/app/routes.py

```diff
@@ -159,8 +154,10 @@ def delete_all():
 
                         file_path = os.path.join(backup_dir, f"bk{i}.json")
 
-                        with open(file_path, "w") as f:
+                        try:
                                 json.dump(data_list, f)
+                        except base64.binascii.Error:
```

Comment:
change the location of the except to be next to the other except. and remove the try as there is one parent whole try